index.js

```diff
@@ -10,6 +10,10 @@ const urlPattern = /^https?:\/\//
 const kWs = Symbol('ws')
 const kWsHead = Symbol('wsHead')
 
+function isDefinedString (string) {
```

Comment:
why :(?

Reply:
Because like this more convenient.
`if (isDefinedString(options.wsUpstream) || isDefinedString(options.upstream))`

Without this function, the condition would have to be written like this:
`if (typeof options.wsUpstream === 'string' && !!options.wsUpstream|| typeof options.upstream === 'string' && !!options.upstream)`

I have a habit of not trusting the input data, I always check for the type.
Or is the question something else?

Reply:
Ok, i will remove the function.

Reply:
Done. Look at last [commit](https://github.com/fastify/fastify-http-proxy/pull/304/commits/0459d2d8eab11a61b9c54b545dc15a4e5d743cae).

Reply:
Can I resolve conversation?